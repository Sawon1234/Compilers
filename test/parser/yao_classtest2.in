class Yao1(xi:Integer) extends P
{
	z:=a-b;
	super(a, b);
	fun hello(): Hello
	{ me:=1;}
}

return 0;