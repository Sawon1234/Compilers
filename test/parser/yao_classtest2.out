class Yao1 < > ( xi : Integer < > ) extends P { z := a . minus < > ( b ) ; super ( a , b ) ; fun hello < > ( ) : Hello < > me := 1 ; } return 0 ;